doc/lua_api.txt

```diff
@@ -229,21 +216,21 @@ Registered names should generally be in this format:
 This is to prevent conflicting names from corrupting maps and is
 enforced by the mod loader.
 
+Enforcement can be overridden by prefixing the name with `:`. This can
```

Comment:
> Enforcement can be overridden

I see what you mean, but in context this seems to say that the A-z0-9_ rules can be overridden, rather than the `modname` naming restriction part.

Reply:
Maybe change it to "Registered names can be overridden...". I moved it out of the examples section because it's not an example.everridden

Reply:
Your suggestion seems ok to me to.